specs/_features/eip7594/das-core.md

```diff
@@ -265,6 +284,26 @@ To custody a particular column, a node joins the respective gossip subnet. Verif
 
 A node SHOULD maintain a diverse set of peers for each column and each slot by verifying responsiveness to sample queries. At each slot, a node makes `SAMPLES_PER_SLOT` queries for samples from their peers via `DataColumnSidecarsByRoot` request. A node utilizes `get_custody_columns` helper to determine which peer(s) to request from. If a node has enough good/honest peers across all rows and columns, this has a high chance of success.
 
+### sample selection
+
+In the worst case scenario (every peer custodies minimum number of columns), the probability that a column is custodied by at least one of the `TARGET_NUMBER_OF_PEERS` is 89.2% (see below calculation), that means the expected number of columns covered by `TARGET_NUMBER_OF_PEERS` is 114, leaving 14 columns not covered.
+
+$$
+E(\text{distinct numbers}) = 128 * P(\text{covered}) = 128 * (1 - P(\text{not covered by at least one peer})) = 128 * (1 - \left( \frac{31}{32} \right)^{70}) = 114
+$$
+
+Therefore it is RECOMMENDED that nodes select sampling columns from what its existing peers custodied instead of from the whole `NUMBER_OF_COLUMNS` range. Compared to random sampling from `NUMBER_OF_COLUMNS` columns, the benefits are:
```

Comment:
I don't think this approach is safe. To get security guarantees your column selection must be strictly random and not influenced by your environment. Otherwise under a peer eclipse an attacker can manipulate your selection